modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/continuous/CacheContinuousQueryHandler.java

```diff
@@ -576,13 +630,48 @@ private String taskName() {
         return mgr.registerListener(routineId, lsnr, internal);
     }
 
+    private CacheContinuousQueryEvent<K,V> transform(CacheContinuousQueryEvent<K, V> evt) {
+        IgniteBiClosure transformer = getTransformer();
```

Comment:
Let's rename to somethink shorter.
For examle "trans", since there is "transVal" variable.